gui/default/syncthing/core/syncthingController.js

```diff
@@ -53,27 +53,30 @@ angular.module('syncthing.core')
         $scope.themes = [];
         $scope.globalChangeEvents = {};
         $scope.metricRates = false;
+        $scope.fsNotificationsAvailable = undefined;
```

Comment:
so {} and [] have different meanings, as they might indicate they are iterable, where as undefined means pretty much nothing.